### DISCUSSION TITLE
Add "type:refactor" issues to generated docs

### DISCUSSION THREAD

**Discussion by setchy** [renovatebot/renovate#21383] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

Seeking feedback on whether it would be useful to have `type:refactor` issues as part of the auto-generated documentation for platforms, managers and data-sources.

Currently, only `type:bug` and `type:feature` items are part of the auto-generated markdown
https://github.com/renovatebot/renovate/blob/e928a8f1b297f60474b7562abe418878833d08a8/tools/docs/github-query-items.ts#L94-L100

**Comment by secustor** [comment]
What would be the scenario that an user needs to know refactoring issues?   
My guess is this will not add viable information, as Contributors can find this information already on Github.

**Reply by setchy** [reply]
_A scenario from this morning..._

This morning I started discussion #21379 which @viceice kindly let me know was an existing refactor ticket - https://github.com/renovatebot/renovate/issues/20961.  

I thought it may be nice on pages like https://docs.renovatebot.com/modules/platform/bitbucket/#open-items to also show key refactoring ideas, so that it's within the curated docs, as well as in the GitHub issues for users who want to slice and dice the 1k open items as they see fit.

Of the 135 currently open [type:refactor](https://github.com/renovatebot/renovate/issues?q=is%3Aopen+is%3Aissue+label%3Atype%3Arefactor issues) issues, there is only a handful that are also associated with `platform:`, `manager:` or `datasource:` label, so it wouldn't add _too much_ new content to the markdown docs

- `platform:*` = 5 issues
- `manager:*` = 11 issues
- `datasource:*`= 10 issues
- others = 109 issues

**Reply by viceice** [reply]
refactor issues are only for development purpose and not for normal users, so they shouldn't be on the docs.

they would confuse more than they are helpful for development

**Reply by setchy** [reply]
Ok, makes sense.  I'll close this suggestion then.
